src/components/Routes/Routes.js

```diff
@@ -35,7 +40,20 @@ export function Routes() {
       );
     }
     return (
-      <Route key={path[0]} path={path[0]} exact component={path[1].component} />
+      <Route key={path[0]} path={path[0]} exact>
```

Comment:
This is now wrapping each route in a `Login` component, not all routes. I'm curious why that change was made? I'm not totally sure why this structure change solves the problem, can you elaborate?

Reply:
A switch statement does not work for nested routes, so I needed to move the <Login> inside each route.

Reply:
Ahh makes sense, thanks